### ISSUE TITLE
Long word is wrapped differently in PDF preview than in PDF generation

### ISSUE THREAD

**Issue by nickiepucel** [pdfme/pdfme#449]
### Describe the bug

When you have a really long, single word in a textbox, the way it is wrapped in the PDF Preview is different than in the PDF when it is generated.

### To Reproduce

This is reproducible in the [Address Label Maker demo app](https://pdfme.com/demo/address-label-maker).

1. Enter the following long string in the first cell in the spreadsheet:

```
123-123-123-123-123-123-123-123-123-123-123-123-123-123-123
```

2.  Observe how the text is wrapped in the Preview
<img width="1277" alt="image" src="https://github.com/pdfme/pdfme/assets/32650952/ecd18dc2-a415-4906-98f1-7e48dd2355a6">
<br>

3. Click Download

4. Notice how the word wrapping appears differently in the generated PDF

<img width="839" alt="image" src="https://github.com/pdfme/pdfme/assets/32650952/aa5c461e-5977-4ab7-b409-52c858548318">

### Expected behavior

The word should be wrapped in the same way in both the preview PDF and generated PDF

### Your Environment

```markdown
- pdfme package(@pdfme/generator or @pdfme/ui):

"@pdfme/common": "^3.2.3",
"@pdfme/generator": "^3.2.3",
"@pdfme/schemas": "^3.2.3",
"@pdfme/ui": "^3.2.3",

- pdfme version: 3.2.3
- Operating system: MacOS Sonoma 14.2.1
- Node.js version or Browser name & version: Chromium 122
```

### Your Error Log

```shell
n/a
```

### Additional context

none

**Comment by peteward** [comment]
Hi,

You do get occassional differences between browser and rendered PDF, it's impossible to get them perfectly the same.

However, in this case I'm wondering if it's because you are using `-`, and the browser might be considering this for a word split where the PDF renderer may not...

**Comment by peteward** [comment]
Comparison of editor mode then PDF using  no spaces (top left), spaces (bottom left), hypens (top-right):

Designer:
![Screenshot 2024-03-20 at 06 21 15](https://github.com/pdfme/pdfme/assets/7068515/94e47349-823e-4bee-a4c8-488bfc6c1925)

PDF:
![Screenshot 2024-03-20 at 06 21 31](https://github.com/pdfme/pdfme/assets/7068515/efd0a3e9-5e19-48e3-a1a4-9a29d5420658)

So it does appear to be specific to hyphens, and [it makes sense when looking at the code](https://github.com/pdfme/pdfme/blob/main/packages/schemas/src/text/helper.ts#L164C1-L166C4).

**Comment by peteward** [comment]
Further tests and it seems that the browser treats hyphen exactly the same as space and so should we. 

Designer
![Screenshot 2024-03-20 at 06 28 09](https://github.com/pdfme/pdfme/assets/7068515/9fdb1cdc-4111-41b2-ace3-cfa8975e641c)

PDF
![Screenshot 2024-03-20 at 06 29 03](https://github.com/pdfme/pdfme/assets/7068515/857c43da-a710-4167-a9f7-0ff4bd80cae7)

I'll write a PR for this soon

**Comment by hand-dot** [comment]
This issue is fixed by #451 
Released as pdfme@3.4.3
